guides/micronaut-data-mongodb-synchronous/micronaut-data-mongodb-synchronous.adoc

```diff
@@ -0,0 +1,129 @@
+common:header.adoc[]
+
+You will use https://www.mongodb.com[MongoDB] for persistence.
+
+common:requirements.adoc[]
+
+common:completesolution.adoc[]
+
+common:create-app-features.adoc[]
+
+=== POJO
+
+Create a `Fruit` POJO:
+
+source:Fruit[]
+
+callout:mapped-entity[1]
+callout:mapped-entity-id[2]
+callout:constraints[3]
+
+=== Repository
+
+Create a repository interface to encapsulate the CRUD actions for `Fruit`.
+
+source:FruitRepository[]
+
+<1> Annotate with `@MongoRepository`.
+<2> Add a finder for finding fruit by a list of names.
+
+=== Service
+
+Create a service `FruitService` to interact with the `Fruit` repository.
+
+source:FruitService[]
+
+callout:singleton[number=1]
+callout:executes-on[number=2]
+
+=== Controller
+
+Create `FruitController`:
+
+source:FruitController[]
+
+callout:controller[number=1,arg0=/fruits]
+callout:constructor-di[number=2,arg0=FruitService]
+callout:get[number=3,arg0=list,arg1=/fruits]
+callout:post[number=4,arg0=save,arg1=/fruits]
+<5> You can specify the HTTP status code via the `@Status` annotation.
+callout:valid[6]
+callout:get[number=7,arg0=find,arg1=/fruits/\{id\}]
+callout:get[number=8,arg0=findByNameInList,arg1=/fruits/q]
+<9> Bind a list of Strings to the query parameter `names`
+
+=== Configuration
```

Comment:
I'd delete the "Configuration" section since the config is the same as what's generated in the application, and this config is discussed below.